backends-velox/src/test/resources/tpch-approved-plan/v1/spark34/9.txt

```diff
@@ -15,31 +15,31 @@ AdaptiveSparkPlan (133)
                                  +- ^ ProjectExecTransformer (80)
                                     +- ^ FlushableHashAggregateExecTransformer (79)
                                        +- ^ ProjectExecTransformer (78)
-                                          +- ^ ShuffledHashJoinExecTransformer Inner (77)
+                                          +- ^ ShuffledHashJoinExecTransformer Inner BuildRight (77)
                                              :- ^ InputIteratorTransformer (68)
                                              :  +- ^ InputAdapter (67)
```

Comment:
Not related to this pr, @zml1206  can you help drop `InputAdapter` in the tree string? It just increases the tree depth. thank you

Reply:
OK, I'll deal with it later.